### PR TITLE
Infrastructure: Check vnu-jar warnings in addition to errors

### PR DESCRIPTION
**Pull request by nschonni**
Few of the other warnings were going to be noisier changes with in-flight PRs, so left them disabled for now

### CODE REVIEW

File: .vnurc [hunk 1]
```diff
@@ -12,3 +12,9 @@ Attribute “aria-posinset” not allowed on element “tr” at this point.
 Attribute “aria-setsize” not allowed on element “tr” at this point.
 # Ignoring role meter
 Bad value “meter” for attribute “role” on element “div”.
+
+# Warnings
```
Comment (spectranaut) [inline .vnurc:16]: Can you make an issue to make sure they are "undisabled" in the future, and leave a link to that issue on this line? :)

**Review by spectranaut — CHANGES_REQUESTED** [review]

File: .vnurc (hunk 1 above)
Comment (nschonni) [reply .vnurc]: Created separate issues and added to file

**Review by spectranaut — APPROVED** [review]
Approved! thanks :)

**Comment by nschonni** [comment]
@mcking65 I gave this a rebase for the conflict

**Review by mcking65 — APPROVED** [review]
Thank you @nschonni; looks good.

**Pull request merged**